RF24.cpp

```diff
@@ -985,6 +1016,608 @@ void RF24::sprintfPrettyDetails(char *debugging_information)
+    // 31 forty_bit_register_array[1]
+    BitManip::packValue(forty_bit_register_array[1], 8, encoded_details, &encoded_details_index, &bit_index);
+    // 32 forty_bit_register_array[2]
+    BitManip::packValue(forty_bit_register_array[2], 8, encoded_details, &encoded_details_index, &bit_index);
+    // 33 forty_bit_register_array[3]
+    BitManip::packValue(forty_bit_register_array[3], 8, encoded_details, &encoded_details_index, &bit_index);
+    // 34 forty_bit_register_array[4]
+    BitManip::packValue(forty_bit_register_array[4], 8, encoded_details, &encoded_details_index, &bit_index);
+
+    // pipe 1 40 bit address register (static_cast<uint8_t>(RX_ADDR_P0 + 1))
+    temp_8_bit = (static_cast<uint8_t>(RX_ADDR_P0 + 1));
+    arrayify_address_register(forty_bit_register_array, temp_8_bit);
+    // 35 forty_bit_register_array[0]
+    BitManip::packValue(forty_bit_register_array[0], 8, encoded_details, &encoded_details_index, &bit_index);
+    // 36 forty_bit_register_array[1]
+    BitManip::packValue(forty_bit_register_array[1], 8, encoded_details, &encoded_details_index, &bit_index);
+    // 37 forty_bit_register_array[2]
+    BitManip::packValue(forty_bit_register_array[2], 8, encoded_details, &encoded_details_index, &bit_index);
+    // 38 forty_bit_register_array[3]
+    BitManip::packValue(forty_bit_register_array[3], 8, encoded_details, &encoded_details_index, &bit_index);
+    // 39 forty_bit_register_array[4]
+    BitManip::packValue(forty_bit_register_array[4], 8, encoded_details, &encoded_details_index, &bit_index);
+
+    // pipe 2 8 bit address register (static_cast<uint8_t>(RX_ADDR_P0 + 2)
+    temp_8_bit = (static_cast<uint8_t>(RX_ADDR_P0 + 2));
+    arrayify_byte_register(eight_bit_register_array, temp_8_bit);
+    // 40 eight_bit_register_array[0]
+    BitManip::packValue(eight_bit_register_array[0], 8, encoded_details, &encoded_details_index, &bit_index);
+
+    // pipe 3 8 bit address register (static_cast<uint8_t>(RX_ADDR_P0 + 3)
+    temp_8_bit = (static_cast<uint8_t>(RX_ADDR_P0 + 3));
+    arrayify_byte_register(eight_bit_register_array, temp_8_bit);
+    // 41 eight_bit_register_array[0]
+    BitManip::packValue(eight_bit_register_array[0], 8, encoded_details, &encoded_details_index, &bit_index);
+
+    // pipe 4 8 bit address register (static_cast<uint8_t>(RX_ADDR_P0 + 4)
+    temp_8_bit = (static_cast<uint8_t>(RX_ADDR_P0 + 4));
+    arrayify_byte_register(eight_bit_register_array, temp_8_bit);
+    // 42 eight_bit_register_array[0]
+    BitManip::packValue(eight_bit_register_array[0], 8, encoded_details, &encoded_details_index, &bit_index);
+
+    // pipe 4 8 bit address register (static_cast<uint8_t>(RX_ADDR_P0 + 5)
+    temp_8_bit = (static_cast<uint8_t>(RX_ADDR_P0 + 5));
+    arrayify_byte_register(eight_bit_register_array, temp_8_bit);
+    // 43 eight_bit_register_array[0]
+    BitManip::packValue(eight_bit_register_array[0], 8, encoded_details, &encoded_details_index, &bit_index);
```

Comment:
Anything against using a for loops?

Reply:
No. It will be nice to look at, easier to read.  I'll push that out now.